MekHQ/data/images/awards/standard/ReadMe.txt

```diff
@@ -0,0 +1,143 @@
+This list takes the Default Set, expands the list, and sorts it in order of precedence as they should appear.  The set is called 'standard' to provide better English when awarding them (i.e. 'Awarded Good Conduct of the standard award set' instead of 'Awarded Good Conduct of the Default Set award set').
+
+This adds 'standard.xml' to ...\MekHQ\data\universe\awards\ and the 'standard' folder to ...\MekHQ\data\images\awards\.
+
+I recommend deleting ...\MekHQ\data\universe\awards\Default Set.xml to remove it from the list of award sets, since it is all duplicated and expanded in the standard set.
+
+The ...\MekHQ\data\images\awards\standard folder also includes a ReadMe.txt with this info as well as a description of all included awards, plus an Excel file titled AwardTracker.xlsx with a PowerPivot Table that can be used to import saved .cpnx campaign files to see what awards have been earned for individuals, lances, companies, and battalions based on kills, as long as your TO&E is arranged by battalions, companies, & lances.
+
+
+Medal of Honor
+	Distinguished oneself conspicuously by gallantry and intrepidity at the risk of life above and beyond the call of duty.
```

Comment:
```suggestion
	Distinguished oneself conspicuously through gallantry and intrepidity at the risk of life above and beyond the call of duty.
```

Reply:
Not certain about this change, but it does sound better when read out.

Reply:
This verbiage is directly from the official MoH. https://en.wikipedia.org/wiki/Medal_of_Honor.  Because of the significance of this award and the respect it deserves, even in a game version, I decided not to change its requirements.  There's only 80 living recipients and 3,500 given out since the Civil War.  There are 1.3 million active members of the US armed forces.  Though I'm concerned about it being taken for granted in a game, I decided to include it for those that have a backstory to tell about one of there characters that would be appropriate.  For this to be earned in-game, someone needs to run a Wasp into the middle of an enemy company to pick up an ejected Mechwarrior, lose a mech limb while being personally injured, and still drop 4 enemy units before successfully escaping to safety.  Something absurd, like rolling 12's four times in a roll with 2D6.